src/components/CardUser/CardUser.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from "next/image";
+import Joined from "@/assets/icons/joined.svg";
+import profile from "@/assets/images/profile.png";
+import classes from "./UserCard.module.css";
+
+interface CardUserProps {
+    width: number;
+    userName: string;
+    status: string;
+}
+export const CardUser = ({ width, userName, status }: CardUserProps & { width: number }) => {
```

Comment:
В `CardUserProps` уже есть `width`

```suggestion
export const CardUser = ({ width, userName, status }: CardUserProps) => {
```

Reply:
Исправил.